src/ScrumBoardItBundle/Services/ApiJira.php

```diff
@@ -43,29 +44,44 @@ public function searchIssues($searchFilters = array())
         
         return array();
     }
-    
-    private function getIssues($data) {
 
+    /**
+     * Return issues based on API results
+     * 
+     * @param \stdClass $data            
+     * @return array
+     */
+    private function getIssues($data)
+    {
         $issues = array();
         foreach ($data->issues as $issue) {
-            if ($issue->fields->issuetype->subtask === true) {
+            if ($issue->fields->issuetype->subtask)
                 $task = new SubTask();
-            } else {
+            else {
                 $task = new Task();
-                $task->setComplexity($issue->fields->customfield_11108);
-                $task->setUserStory(true);
+                if (! empty($issue->fields->customfield_11108)) {
+                    $task->setComplexity($issue->fields->customfield_11108);
```

Comment:
customfield_11108 must be in configuration (because it depends on target jira configuration fields).